test_scripts/API/VehicleData/UnsubscribeVehicleData/005_Success_flow_2_apps.lua

```diff
@@ -0,0 +1,135 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TO ADD !!!
+-- Use case: TO ADD !!!
+-- Item: Use Case: request is allowed by Policies
+--
+-- Requirement summary:
+-- [UnsubscribeVehicleData] Mobile app wants to send a request to unsubscribe
+--  for already subscribed specified parameter
+--
+-- Description:
+-- In case:
+-- App1 is alredy subscribed to vehicle data
+-- App2 is alredy subscribed to vehicle data
+-- App2 sends valid UnsubscribeVehicleData to SDL
+-- This request is allowed by Policies and mobile app is subscribed for this parameter
```

Comment:
@GetmanetsIrina , typo 
```suggestion
-- This request is allowed by Policies and mobile app is unsubscribed for this parameter
```